technical-resources/singletons/globals.md

```diff
@@ -4,7 +4,7 @@ MapleGlobals is a singleton contract used to save system-wide parameters that al
 
 # Permissioning
 
-The governor multisig is the only entity allowed to change the parameters stored in MapleGlobals, the only exception is pause-related parameters which can also be done by the Security Admin.
+The governor multisig holds the primary authority to change parameters stored in MapleGlobals. Additionally, the operational admin, authorized by the Governor, can change specific operational parameters and Security Admin can change pause-related parameters.
```

Comment:
`The governor multisig is primarily responsible for changing parameters stored in MapleGlobals.`
`...and the Security Admin can change parameters related to pausing.`

Reply:
I think the sentence is already in simple present tense and conveys the info.

Reply:
used chat gpt